### PR TITLE
Adds help message to AI core

### PR DESCRIPTION
**Pull request by TealSeer**
[QOL] [UI]
## About the PR 

Adds a dynamic help message to AI cores explaining each tool that can be used on it. 

## Why's this needed? 

I never remember these

### CODE REVIEW

File: code/mob/living/silicon/ai.dm [hunk 1]
```diff
@@ -129,6 +129,7 @@ var/global/list/ai_emotions = list("Annoyed" = "ai_annoyed-dol", \
 	var/status_message = null
 	var/mob/living/silicon/deployed_shell = null
 	var/locking = 0
+	HELP_MESSAGE_OVERRIDE(null)
```
Comment (TobleroneSwordfish) [inline code/mob/living/silicon/ai.dm:132]:
```suggestion
```
Don't think this is necessary?

**Review by TobleroneSwordfish — APPROVED** [review]

File: code/mob/living/silicon/ai.dm [hunk 2]
```diff
@@ -245,6 +246,20 @@ or don't if it uses a custom topopen overlay
...
+		if(0)
+			. = "You can swipe an <b>ID card</b> to unlock the cover."
+		if(1)
+			. = "You can use a <b>crowbar</b> to pry open the cover, or swipe an <b>ID card</b> to lock it."
+		if(2)
+			. = "You can use a <b>wrench</b> to undo the CPU bolts, <b>cable coil</b> to repair damage, or a <b>crowbar</b> to close the cover."
+		if(3)
+			. = "You can use a <b>wrench</b> to tighten the CPU bolts, or an <b>empty hand</b> to remove the CPU unit."
+		if(4)
+			. = "You can insert a <b>brain</b> to activate the AI."
```
Comment (444Portal) [inline code/mob/living/silicon/ai.dm:260]: this doesn't mention the fact you need to 'restart' it for the AI to work properly - they'll ghost right after unless restarted

File: code/mob/living/silicon/ai.dm (hunk 1 above)
Comment (TealSeer) [reply code/mob/living/silicon/ai.dm:132]: you need that for the Help verb to appear in the right click menu, it doesn't if you just override `get_help_message`

File: code/mob/living/silicon/ai.dm [hunk 3]
```diff
@@ -245,6 +246,20 @@ or don't if it uses a custom topopen overlay
...
+		if(0)
+			. = "You can swipe an <b>ID card</b> to unlock the cover."
+		if(1)
+			. = "You can use a <b>crowbar</b> to pry open the cover, or swipe an <b>ID card</b> to lock it."
+		if(2)
+			. = "You can use a <b>wrench</b> to undo the CPU bolts, <b>cable coil</b> to repair damage, or a <b>crowbar</b> to close the cover."
+		if(3)
+			. = "You can use a <b>wrench</b> to tighten the CPU bolts, or an <b>empty hand</b> to remove the CPU unit."
+		if(4)
+			. = "You can insert a <b>brain</b>, followed by an <b>empty hand</b> to reboot the AI."
```
Comment (TobleroneSwordfish) [inline code/mob/living/silicon/ai.dm:260]: Ideally the instruction to reboot the AI should show when the brain is in and the AI needs rebooting, not before you insert it. Currently putting a brain in an inactive AI just shows options to tighten the bolts or remove the brain again.

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

File: code/mob/living/silicon/ai.dm [hunk 4]
```diff
@@ -256,6 +256,8 @@ or don't if it uses a custom topopen overlay
 			. = "You can use a <b>wrench</b> to undo the CPU bolts, <b>cable coil</b> to repair damage, or a <b>crowbar</b> to close the cover."
 		if(3)
 			. = "You can use a <b>wrench</b> to tighten the CPU bolts, or an <b>empty hand</b> to remove the CPU unit."
+			if(isdead(src))
+				. += " You can use an <b>empty hand</b> to reboot the AI."
```
Comment (TobleroneSwordfish) [inline code/mob/living/silicon/ai.dm:260]: You can do this at any deconstruction stage, not just 3

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request merged**